### PR TITLE
Add list item and sources dialog to yadocs form

### PR DESCRIPTION
**Pull request by korvin89**
<details>
<summary>Add this part of UI:</summary>

ololo2345

</details>

### CODE REVIEW

File: src/ui/units/connections/components/custom-forms/Yadocs/components/DocsList.tsx [hunk 1]
```diff
@@ -1,31 +1,56 @@
...
+    const {
+        addSectionState,
+        items,
+        selectedItemIndex,
+        addYandexDoc,
+        clickListItem,
+        updateAddSectionState,
+    } = props;
+
+    const renderItem = React.useCallback((item: YadocListItem) => {
```
Comment (mournfulCoroner) [inline src/ui/units/connections/components/custom-forms/Yadocs/components/DocsList.tsx:32]: Is there a reason why this function should be inside the component and not outside?

Comment (korvin89) [reply src/ui/units/connections/components/custom-forms/Yadocs/components/DocsList.tsx:32]: I suppose it's pretty clear to keep items render method near List component.
There are store actions outside this component. This method handle UI

File: src/ui/units/connections/components/custom-forms/Yadocs/components/YadocListItemView/UploadedYadocView.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import {I18n} from 'i18n';
+import {parseError} from 'ui/utils/errors/parse';
+
+import {ConverterErrorCode} from '../../../../../constants';
+import type {UploadedYadoc} from '../../../../../store';
+import {ListItem, ListItemAction} from '../../../components';
+// import type {HandleItemClick} from '../../types';
```
Comment (mournfulCoroner) [inline src/ui/units/connections/components/custom-forms/Yadocs/components/YadocListItemView/UploadedYadocView.tsx:9]: Are these the forgotten comments here and below?

**Review by mournfulCoroner — DISMISSED** [review]

File: src/ui/units/connections/components/custom-forms/Yadocs/components/YadocListItemView/UploadedYadocView.tsx (hunk 2 above)
Comment (korvin89) [reply src/ui/units/connections/components/custom-forms/Yadocs/components/YadocListItemView/UploadedYadocView.tsx:9]: Yep. I'll fix it

**Review by mournfulCoroner — APPROVED** [review]

**Review by mournfulCoroner — APPROVED** [review]

**Pull request merged**